### DISCUSSION TITLE
migration tailwind(twin.macro) to panda

### DISCUSSION THREAD

**Discussion by p-iknow** [chakra-ui/panda#763] [Q&A]
## background 

Currently, our team is using next.js, typescirpt, emotion. twin.macro(tailwind). We are satisfied with `twin.macro`, but `twin.macro` depends on babel macro, which is difficult to use in an environment where babel is not supported. Because of this, our team has a problem that we cannot use SWC environment of next.  We tried to use a library called stailwc to solve this problem, but it is still too early to use it at the production level. Therefore, we are going to use panda.css, which has some advantages. 

## What I want to accomplish 

To ease the migration, the set of utility names used in the existing tailwind classNames should be available in panada.css. 

Since panda.css also provides [uiliity]( https://panda-css.com/docs/customization/utilities), maybe we could do this by setting the same set of utils used in [tailwind](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/config.full.js).

## Any plan to make a tool to help migration?
I'm considering migrating with the above plan. Is there a better way to do it, or do you have any advice?
Or are you planning something to ease the migration from the existing css in js library to panda?

**Comment by anubra266** [comment]
@p-iknow I think the way around this will be to create a community twind preset for panda, so you can just add it to your panda config and have that all ready.
